### ISSUE TITLE
Put in redirects for old E&J appendix pages

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#5515]
### Summary

**What we're after:**
_As a user who wants to research old versions of regulations, I need the revised E&J appendix pages._

Once the updated E&J appendix pages are migrated, we'll need to redirect the old pages to the new ones.

### Related issues

- #5426 

### Completion criteria

- [ ] Redirects for old E&J pages are created and merged
- [ ] Redirect spreadsheet is updated
- [ ] Testing ticket is created

### Future work

Once redirects are tested and working, we'll add the E&J appendix pages to a take-down ticket.

**Comment by dorothyyeager** [comment]
Redirect PR has been submitted at https://github.com/fecgov/fec-proxy/pull/349.

Once that is approved and merged, this ticket can be closed. Testing ticket created at https://github.com/fecgov/fec-cms/issues/5556.
